src/ui/main.cpp

```diff
@@ -94,7 +94,6 @@ int main(int argc, char *argv[])
     // There are no other instances: start a new server.
     a.startServer();
 
-    Editor::addEditorToBuffer();
```

Comment:
Oh, got it!

By the way, I've just pushed the new nqq version with your changes to the ppa. The updated packages will become available in a few minutes ;)